src/corespotlight.cs

```diff
@@ -74,6 +74,15 @@ interface CSSearchableIndex {
 		[Export ("initWithName:protectionClass:")]
 		NativeHandle Constructor (string name, [NullAllowed] NSString protectionClass);
 
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[iOS (9, 0), Mac (10, 11), MacCatalyst (13, 1), NoTV, NoWatch]
+		[Export ("initWithName:protectionClass:bundleIdentifier:options:")]
+		NativeHandle Constructor (string name, [NullAllowed] string protectionClass, string bundleIdentifier, nint options);
```

Comment:
I think you want instead
```suggestion
		NativeHandle Constructor (string name, [NullAllowed] NSString protectionClass, string bundleIdentifier, nint options);
```